experiments/conductor/cmd/runner/mock_commands.go

```diff
@@ -171,28 +179,31 @@ func captureHttpLog(opts *RunnerOptions, branch Branch) {
 		return
 	}
 
+	// Enable required APIs before running tests
+	if err := enableAPIs(opts, branch); err != nil {
```

Comment:
Would it make more sense to just make enableAPIs a top level command? Then you call it once per project rather than every time?

Reply:
done added a command to do that. 
Had to move commands over to new numbers.